Snakefile

```diff
@@ -25,207 +25,229 @@ f.close()
 #get seeds and hemis from config
 seeds = config['seeds']
 hemis = config['hemis']
-
+excrois = config['excrois']
 
 wildcard_constraints:
-    subject="[a-zA-Z0-9]+",
-    template="[a-zA-Z0-9]+"
+subject="[a-zA-Z0-9]+",
+template="[a-zA-Z0-9]+"
 
 
 rule all:
-    input: 
-        clusters = expand('diffparc/clustering/group_space-{template}_seed-{seed}_hemi-{hemi}_method-spectralcosine_k-{k}_cluslabels.nii.gz',seed=seeds,hemi=hemis,template=config['template'],k=range(2,config['max_k']+1))
+	input: 
```

Comment:
your editor seems to be using tabs for indents instead of spaces -- spaces (4spaces for an indent) is the python standard so should stick with that -- what editor are you using? should be able to force it to do this..

Reply:
I am using Microsoft Visual Studio  - yes noticed that and was trying to fix... will search if there's any shortcut / setting...